bazelrio/scripts/deploy/Deploy.java

```diff
@@ -61,6 +63,9 @@ public static void main(String[] args) throws IOException {
             .type(String.class)
             .nargs("*")
             .setDefault();
+        parser.addArgument("--fast_unsafe_deploy")
```

Comment:
I don't know that I love this name. Maybe `skip_shared_libs`, `skip_so` `skip_dll` etc. 'unsafe` smells sketchy

Reply:
Also, a a description about what it does, i.e. assume all the DLL's have already been transfered